pyproject.toml

```diff
@@ -86,7 +91,10 @@ plugins = ["pydantic.mypy"]
 [tool.ruff]
 line-length = 120
 
+[tool.poetry.scripts]
+humanloop = "humanloop.cli.__main__:cli"
```

Comment:
integration test: use subprocess to check if cli works

Reply:
check if exit code 0 when calling cli command; maybe check for exit code 1 when api key not provided